src/portal/src/lib/components/image-name-input/image-name-input.component.ts

```diff
@@ -71,7 +93,7 @@ export class ImageNameInputComponent implements OnInit, OnDestroy {
                 } else {
                     this.errorHandler.error("not Observable type");
                 }
-            });
+            });*/
```

Comment:
Why comment out the old code? Not remove them?

Reply:
fixed